view/adminhtml/templates/sales_order_create_customer_block_script.phtml

```diff
@@ -1,17 +1,23 @@
 <?php
-    $pcaSettings = $this->helper('PCAPredict\Addressy\Helper\SettingsData');
-    $pcaAccCode = $pcaSettings->getAccountCode();
-    $pcaCustomJSBack = $pcaSettings->getCustomJavaScriptBack();
+/**
+ * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
+ * @var \Magento\Framework\View\TemplateEngine\Php $this
+ */
 
-    if ($pcaAccCode) { ?>
+$pcaSettings = $this->helper('PCAPredict\Addressy\Helper\SettingsData');
+$pcaAccCode = $pcaSettings->getAccountCode();
+$pcaCustomJSBack = $pcaSettings->getCustomJavaScriptBack() ?: '';
 
-<script>
+if (!$pcaAccCode) {
+    return;
+}
 
+$script = <<<JS
     (function (a, c, b, e) {
-    a[b] = a[b] || {}; a[b].initial = { accountCode: "<?php echo $pcaAccCode; ?>", host: "<?php echo $pcaAccCode; ?>.addressy.com" };
+    a[b] = a[b] || {}; a[b].initial = { accountCode: "{$pcaSettings}", host: "{$pcaAccCode}.addressy.com" };
```

Comment:
```suggestion
    a[b] = a[b] || {}; a[b].initial = { accountCode: "{$pcaAccCode}", host: "{$pcaAccCode}.addressy.com" };
```
Seems like this was a typo?

Reply:
🙀 wow, thanks for finding that. It's odd, the address typeahead kept working